### ISSUE TITLE
Issue with k8s.io/docs/tasks/access-application-cluster/ingress-minikube/

### ISSUE THREAD

**Issue by FredericLatour** [kubernetes/website#16503]
**This is a Bug Report**

I'm following the instructions on the page and and `curl hello-world.info` doesn't give the expected result.

**Problem:**
After following the instructions, when I `curl hello-world.info`, I get the following result:

```bash
$ curl -k hello-world.info
<html>
<head><title>404 Not Found</title></head>
<body>
<center><h1>404 Not Found</h1></center>
<hr><center>openresty/1.15.8.1</center>
</body>
</html>
```
I can confirm that the deployment and service work correctly. Accessing the service via:
`minikube service web --url
`
output the expected result (as described in the documentation).

**Proposed Solution:**
No idea but I have a hard time to understand how it can be so difficult to set up a kubernetes dev env with minikube.
No one minikube/ingress example I could find seem to work properly, at least on my minikube installation. 

**Page to Update:**
https://kubernetes.io/...

**Kubernetes Version:**
Client Version: version.Info{Major:"1", Minor:"14", GitVersion:"v1.14.6", GitCommit:"96fac5cd13a5dc064f7d9f4f23030a6aeface6cc", GitTreeState:"clean", BuildDate:"2019-08-19T11:13:49Z", GoVersion:"go1.12.9", Compiler:"gc", Platform:"windows/amd64"}
Server Version: version.Info{Major:"1", Minor:"15", GitVersion:"v1.15.2", GitCommit:"f6278300bebbb750328ac16ee6dd3aa7d3549568", GitTreeState:"clean", BuildDate:"2019-08-05T09:15:22Z", GoVersion:"go1.12.5", Compiler:"gc", Platform:"linux/amd64"}

**Comment by sftim** [comment]
The current page suggests

> Add the following line to the bottom of the `/etc/hosts` file.
> Note: If you are running Minikube locally, use minikube ip to get the external IP. The IP address displayed within the ingress list will be the internal IP.

It sounds possible to clarify that, helping readers clearly understand what steps to follow so that the subsequent `curl` command works OK.

/priority backlog

**Comment by FredericLatour** [comment]
@sftim I should had mentioned that I had indeed updated my Hosts file. I was able to ping hello-world.info and get a reponse from minikube ip.
And yes, I'm aware of the `minikube ip` command.
Just to be sure, I will re-play the page instructions once more.

**Comment by sftim** [comment]
@FredericLatour if you can clarify what needs to change on the page to get it right, that'll help a lot towards a fix.

**Comment by FredericLatour** [comment]
@sftim It works now. It seems that the page was updated today and that the reason it was not working was because there was no base path. (that is `hello-world.info/whatever` would have worked properly)
On a different note: do you know why when I access hello-world ingress does not seem to automatically redirect to https whereas it is the case each time I tried to set up a site of mine?

**Comment by sftim** [comment]
Thanks for the update @FredericLatour 
You can get more help via [Stack Overflow](https://stackoverflow.com/questions/tagged/kubernetes), from the [Kubernetes Slack](http://slack.k8s.io/) organization, or the [Discuss Kubernetes](https://discuss.kubernetes.io/) forum.

Fixed via #16458
/close
